<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRSPlannerWindow</class>
 <widget class="QMainWindow" name="CRSPlannerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot-Airship Facility  -  Motion Planner</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Positions"/>
    <addaction name="actionSave_Positions"/>
    <addaction name="actionLoad_Planner"/>
    <addaction name="actionSave_Planner"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun_Dialog"/>
    <addaction name="actionRun_Planner"/>
    <addaction name="actionStart_Sol_Anim"/>
    <addaction name="actionStop_Sol_Anim"/>
    <addaction name="actionStart_Target_Anim"/>
    <addaction name="actionStop_Target_Anim"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Positions">
   <property name="text">
    <string>Load Positions...</string>
   </property>
  </action>
  <action name="actionSave_Positions">
   <property name="text">
    <string>Save Positions...</string>
   </property>
  </action>
  <action name="actionLoad_Planner">
   <property name="text">
    <string>Load Planner...</string>
   </property>
  </action>
  <action name="actionSave_Planner">
   <property name="text">
    <string>Save Planner...</string>
   </property>
  </action>
  <action name="actionRun_Planner">
   <property name="text">
    <string>Run Planner</string>
   </property>
  </action>
  <action name="actionStart_Sol_Anim">
   <property name="text">
    <string>Start Sol. Anim.</string>
   </property>
  </action>
  <action name="actionStop_Sol_Anim">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop Sol. Anim.</string>
   </property>
  </action>
  <action name="actionStart_Target_Anim">
   <property name="text">
    <string>Start Target Anim.</string>
   </property>
  </action>
  <action name="actionStop_Target_Anim">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop Target Anim.</string>
   </property>
  </action>
  <action name="actionRun_Dialog">
   <property name="text">
    <string>Run Dialog...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
